resources/schema.py

```diff
@@ -490,6 +490,17 @@ def mutate_and_get_payload(cls, root, info, **input):
 class HarborInput(AbstractAreaInput):
     municipality_id = graphene.String()
     image_file = Upload()
+    add_map_files = graphene.List(
+        Upload,
+        description="List of map files that will be added to the existing ones belonging to the Harbor.",
+    )
+    remove_map_files = graphene.List(
```

Comment:
I think a cleaner solution would be to add this input field on `UpdateHarborMutation`'s `Input` directly. It feels odd, that we have a field user can pass value in, but any value would cause an error.

Reply:
Makes more sense. 👍